### ISSUE TITLE
backup not generated automatically

### ISSUE THREAD

**Issue by abilashks** [sameersbn/docker-gitlab#538]
I am running gitlab docker and its working fine. but backup tar is not generated.
below is my command
sudo docker run --restart=always --name gitlab_8_0_5_1_cm2-test -d  --env 'TZ=Asia/Kolkata' --env 'GITLAB_HOST=git' --env "USERMAP_UID=$(id -u git)" --env "USERMAP_GID=$(id -g git)" --env 'REDIS_HOST=redis-host' --env 'REDIS_PORT=6379' --env 'NGINX_WORKERS=8' --env 'UNICORN_WORKERS=16' --env  'DB_TYPE=mysql' --env 'GITLAB_BACKUPS=daily' --env 'GITLAB_BACKUP_TIME=10:00'  --env 'GITLAB_BACKUP_DIR=/home/git/data/backups'  --env 'GITLAB_EMAIL=git@gmail.com' --env 'SMTP_ENABLED=true' --env 'SMTP_HOST=smtp' --env 'SMTP_PORT=25' --volume /home/git//gitlab:/home/git/data -v /home/git//gitlab/logs:/var/log/gitlab/gitlab -v /home/git/gitlab/hooks:/home/git/gitlab-shell/hooks -p 2023:22 -p 8081:80 --env 'DB_TYPE=mysql' --env 'DB_HOST=mysql-host' --env 'DB_NAME=gitlabhq_production' --env 'DB_USER=git' --env 'DB_PASS=mysqlgitpass' --env 'DB_POOL=50' --env 'DB_PORT=3307' --env 'GITLAB_SECRETS_DB_KEY_BASE=gitlab-key' --env 'GITLAB_CI_SECRETS_DB_KEY_BASE=gitlab-ci-key' --env 'GITLAB_GRAVATAR_ENABLED=false' --env 'GITLAB_TIMEZONE=Chennai' sameersbn/gitlab:8.0.5-1

Please help

**Comment by abilashks** [comment]
i get below error when i try to run manually
Running "gitlab:backup:create" rake task ...
Dumping database ... 
Dumping MySQL database gitlabhq_production ... mysqldump: Error 2020: Got packet bigger than 'max_allowed_packet' bytes when dumping table `merge_request_diffs` at row: 16750
[FAILED]

but i have set max_allowed_packet = 2048M in my my.cnf file of db

**Comment by sameersbn** [comment]
@abilashks well the error says what it says. you need to handle this in your mysql configuration. Errors such as there are not uncommon when you choose to you mysql with gitlab. I have no suggestion on how to fix this other than increasing the `max_allowed_packet` further.

Users who faced this issue and got it resolved may be able to help you.

**Comment by abilashks** [comment]
yes the issue got fixed by increasing max_allowed_packet in gitlab container. But this has to be done manually after invoking the container. We have to do it for each versiion upgrade. It will be great if you could provide this as a option.
